app/service/contact_svc.py

```diff
@@ -67,10 +67,26 @@ def __init__(self, agent_config):
         self._sleep_min = agent_config['sleep_min']
         self._sleep_max = agent_config['sleep_max']
         self._watchdog = agent_config['watchdog']
-        self._file_names = agent_config['names']
+        self._file_names = self.collect_file_names(agent_config['names'])
         self._untrusted_timer = agent_config['untrusted_timer']
         self._bootstrap_instructions = agent_config['bootstrap_abilities']
 
+    def collect_file_names(self, names):
+        multiplatform = []
```

Comment:
I think you can just change what calls self.-file-names instead of this

Reply:
I don't think so - not if we are enabling comma-separated platform definitions (`linux,freebsd:`)